webextensions/api/browsingData.json

```diff
@@ -16,8 +16,7 @@
                   "version_added": "53"
                 },
                 "firefox_android": {
-                  "version_added": "56",
-                  "version_removed": "79"
+                  "version_added": "85"
```

Comment:
Was the previous data here wrong? Or did it actually happen that it was implemented in Fennec and then appeared in Fenix in version 85? If it's the latter, then we should structure this data as an array of support statements:

```
[
  {
     "version_added": "85"
  },
  {
     "version_removed": "79",
     "version_added: "56"
  }
]
```

(this would apply to the other instances like this in the diff, as well)

Reply:
It was implemented in Fennec and now we are re-implementing it in Fenix. Thanks for your comment! I updated the patch.